src/__configuration__/contributors/contributors.ts

```diff
@@ -106,4 +107,15 @@ export const contributors: Contributor[] = [
         description: 'Icons',
         image: RenatoMartins,
     },
+
+    {
+        name: 'Joseph Boyle',
+        description: `Front-End Developer & Strategy`,
+        image: JosephBoyle,
+        info: `Forsaking all others, Joe has been with PX Blue since its inception. A double-threat with a background in computer science and user experience, he's not afraid to drop 100+ comments on your most recent PR. Befriend at your own risk.`,
```

Comment:
How are the fields `info` & `contacts` used here?

Reply:
Can go either way; they share the same type, but info and contacts don't get rendered

Reply:
I'm thinking these should be two different types if those other fields aren't used.